pkg/config/distinguished-name.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2023 LY Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
```

Comment:
I still do not understand why non `config` methods are under the directory `config` instead of `utils` or anything else. but not a strong opinion and definitely OK if you think it `config` is the right choice in this PR.

Reply:
And technically this file should be inside the `dervied-cert-subject.go` at this point, so please give a look once again.

Reply:
merged `distinguished-name.go` and `dervied-cert-subject.go` as single file